code/modules/factory/parts.dm

```diff
@@ -693,6 +701,23 @@ GLOBAL_LIST_INIT(thermobaric_wp_recipe, list(
 	list(STEP_NEXT_MACHINE = FACTORY_MACHINE_GALVANIZER, STEP_ICON_STATE = "rockettube"),
 	))
 
+/obj/item/factory_part/mlrs_rocket/gas
+	name = "\improper MLRS mustard gas rocket"
+	desc = "An unfinished mustard gas rocket."
+	result = /obj/item/storage/box/mlrs_rockets/gas
+
+/obj/item/factory_part/mlrs_rocket/cloak
+	name = "\improper MLRS cloak rocket"
+	desc = "An unfinished cloak gas rocket."
+	result = /obj/item/storage/box/mlrs_rockets/cloak
+
+/obj/item/factory_part/mlrs_rocket/incendiary
+	name = "\improper MLRS Incendiary rocket"
+	desc = "An unfinished incendiary rocket."
+	result = /obj/item/storage/box/mlrs_rockets/incendiary
+
```

Comment:
this should be before the thermo recipy not after, lgtm otherwise

Reply:
Done